core/src/com/csc14/runtimeterrors/game/OCClient.java

```diff
@@ -248,4 +248,35 @@ public OCMessage declineInvite(String user, String inviter){
         return receivedMessage;
     }
 
+    // request the matches a user can resume
+    public OCMessage getResumeMatches(String nickname) {
+        System.out.println("Sending get in-progress matches request for " + nickname);
+
+        OCMessage message = new OCMessage();
+        message.put("process", "get in-progress matches");
+        message.put("nickname", nickname);
+
+        // receive message
+        OCMessage receivedMessage = sendRequestAndReceiveMessage(message);
+
+        printResult(receivedMessage);
+
+        return receivedMessage;
+    }
+
+    // request the matches a user can resume
```

Comment:
Should this be "request to resume a match"?

Reply:
should have but request was not needed